src/Visualization/Python/PlotTrajectories.py

```diff
@@ -0,0 +1,179 @@
+
+import logging
+
+import click
+import h5py
+import matplotlib.pyplot as plt
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+from spectre.Visualization.Plot import (
+    apply_stylesheet_command,
+    show_or_save_plot_command,
+)
+
+
+@click.command(name="trajectories")
+@click.argument(
+    "h5_files",
+    nargs=-1,
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+)
+@click.option(
+    "--subfile-name-aha",
+    "-A",
+    default="ApparentHorizons/ControlSystemAhA_Centers.dat",
+    help=(
+        "Name of subfile containing the apparent horizon centers for object A."
+    ),
+)
+@click.option(
+    "--subfile-name-ahb",
+    "-B",
+    default="ApparentHorizons/ControlSystemAhB_Centers.dat",
+    help=(
+        "Name of subfile containing the apparent horizon centers for object B."
+    ),
+)
+@click.option(
+    "--sample-rate",
+    "-sr",
+    type=int,
+    default=15,
+    show_default=True,
+    help="Downsampling value for 3D plots",
+)
+@click.option(
+    "--figsize",
+    nargs=2,
+    type=float,
+    default=(10.0, 10.0),
+    show_default=True,
+    help="Figure size as width and height in inches",
+)
+@apply_stylesheet_command()
+@show_or_save_plot_command()
+def plot_trajectories_command(
+    h5_files, subfile_name_aha, subfile_name_ahb, sample_rate, figsize=(10, 10)
+):
+    """Plot trajectories in inspiral simulation
+
+    Concatenates partial trajectories from each h5 reductions files and plots
+    full trajectories.
+    """
+    reductionfiles = h5_files
+    AhA, AhB = import_A_and_B(
+        reductionfiles, subfile_name_aha, subfile_name_ahb
+    )
+    plot_trajectory(AhA, AhB, sample_rate, figsize)
+
+
+def import_A_and_B(filenames, subfile_name_aha, subfile_name_ahb):
+    A_data = []
+    B_data = []
+
+    for filename in filenames:
+        with h5py.File(filename, "r") as file:
+            A_data.append(
+                np.array(
+                    file[subfile_name_aha][:, [4, 5, 6]]
+                )  # 0 ->time,  4 -> x, 5 -> y, 6 -> z
+            )
+            B_data.append(
+                np.array(
+                    file[subfile_name_ahb][:, [4, 5, 6]]
+                )  # 0 ->time,  4 -> x, 5 -> y, 6 -> z
+            )
+
+    if A_data and B_data:
+        A = np.concatenate(A_data)
+        B = np.concatenate(B_data)
+        min_length = min(len(A), len(B))
+        return A[:min_length], B[:min_length]
+    else:
+        return None, None
+
+
+def plot_trajectory(AhA, AhB, sample_rate, figsize):
+    """Plot concatenated trajectories in inspiral simulation
+
+    Outputs a 2x2 figure with 4 plots:
+    1st row: A and B trajectories in 3D, coordinate separation in 3D
+    2nd row: A and B trajectories in 2D xy plane, coordinate separation in 2D
+    xy plane. the 3D plots use a subsampled version of the data, with default
+    value of 15, to speed up the plots and avoid memory error
```

Comment:
Add a description of the expected arguments, like this:

```
Arguments:
  AhA: Array of shape (num_points, 3) with the coordinates of the first object.
  AhB: ...
  sample_rate: ...
  figsize: ...
```

This is important so people know how to use your function.